edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/modules/RoutingModule.cs

```diff
@@ -577,11 +580,22 @@ protected override void Load(ContainerBuilder builder)
                                 this.versionInfo,
                                 deviceScopeIdentitiesCache);
 
-                            return new TwinConfigSource(edgeHubConnection, edgeHubCredentials.Identity.Id, this.versionInfo, twinManager, twinMessageConverter, twinCollectionMessageConverter, routeFactory);
+                            return new TwinConfigSource(
+                                edgeHubConnection,
+                                edgeHubCredentials.Identity.Id,
+                                this.versionInfo,
+                                twinManager,
+                                twinMessageConverter,
+                                twinCollectionMessageConverter,
+                                routeFactory);
                         }
                         else
                         {
-                            return new LocalConfigSource(routeFactory, this.routes, this.storeAndForwardConfiguration);
+                            return new LocalConfigSource(
+                                routeFactory,
+                                this.routes,
+                                this.storeAndForwardConfiguration,
+                                this.authorizationConfiguration);
```

Comment:
The default behavior here is what? I looked and a new AuthorizationConfiguration is empty, so does this mean that everything is denied?